### DISCUSSION TITLE
Editing .yml file so only cam0 runs during certain hours and cam1 runs during certain hours, both cameras never running at same time

### DISCUSSION THREAD

**Discussion by HelpMePlsOkNow** [bluenviron/mediamtx#3990] [Questions]
### Question

Hello, I'm trying to have cam0 run only daytime and cam1 only run during nighttime, is this possible through editing .yml file?  The Readme doesn't seem to cover this,  Thanks in advance for your help.  Current edits are below:

paths:
  cam1:
    runOnInit: bash -c 'rpicam-vid -t 0 --camera 0 --nopreview --codec yuv420 --width 1280 --height 720 --inline --listen -o - | ffmpeg -f rawvideo -pix_fmt yuv420p -s:v 1280x720 -i /dev/stdin -c:v libx264 -pre>
    runOnInitRestart: yes
    runOnReady: >
      gst-launch-1.0
      rtspclientsink name=s location=rtsp://localhost:$RTSP_PORT/cam_with_audio
      rtspsrc location=rtsp://127.0.0.1:$RTSP_PORT/$MTX_PATH latency=0 ! rtph264depay ! s.
      alsasrc device=hw:CARD=II,DEV=0 ! opusenc bitrate=16000 ! s.
    runOnReadyRestart: yes
  cam_with_audio:
  # example:
  # my_camera:
  #   source: rtsp://my_camera

  # Settings under path "all_others" are applied to all paths that
  # do not match another entry.
  all_others:

**Comment by dbozz** [comment]
I'm still not very familiar with the config of Mediamtx, but something that I think should work is using the Hot Reloading feature. 
Just create an external script that runs with cron and update the config file accordingly every morning/evening.

**Reply by HelpMePlsOkNow** [reply]
Ok, yes, I was thinking about that as well, I think it will be easier just to have both cameras running, thank you for the reply.
